tests/unit/clients/python/test_io.py

```diff
@@ -73,12 +73,40 @@ def test_input_lines_with_empty_filepath_and_lines():
         for _ in lines:
             pass
 
+@pytest.mark.parametrize(
+    'size, sampling_rate',
+    [
+        (None, None),
+        (1, None),
+        (None, 0.5),
+    ],
+)
+def test_input_lines_with_jsonlines_file(size, sampling_rate):
+    result = list(from_lines(filepath='tests/unit/clients/python/docs.jsonlines', size=size, sampling_rate=sampling_rate))
+    assert len(result) == size if size is not None else 2
+    if sampling_rate is None:
```

Comment:
Can you try to move the expectation for the assert into the parametrize part? I think that makes it easier to read.
With the nested if here its a bit hard to understand what is happening and why.
Some like here I mean: https://github.com/jina-ai/jina-hub/blob/9fba74c85cd8c4c03655cfebbd4031de0cb1d53c/evaluators/rank/AveragePrecision/tests/test_averageprecisionevaluator.py#L20